sklego/meta/outlier_classifier.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.utils.validation import (
+    check_is_fitted,
+    check_X_y
+)
+
+
+class OutlierClassifier(BaseEstimator, ClassifierMixin):
+    """
+    Morphs an estimator that performs outlier detection into a classifier.
+    When an outlier is detected it will output 1 and 0 otherwise.
+    This way you can use familiar metrics again and this allows you
+    to consider outlier models as a fraud detector.
+    """
+    def __init__(self, model):
+        self.model = model
+
+    def _is_outlier_model(self):
```

Comment:
Would this be solved in a nicer way by implementing another meta model, like the `ProbabilisticClassifier`


```class OutlierModelMeta(type):
    def __instancecheck__(self, other):
        return isinstance(self.model, OutlierMixin)

class OutlierModel(metaclass=OutlierModelMeta):
    pass
```

Reply:
Valid. Made the changes.